### PR TITLE
Task 45 software guidebook aanvullen

### PR DESCRIPTION
**Pull request by StefanoudeLohuis**

### CODE REVIEW

**Review by ilivss — APPROVED** [review]
Goed I guess

**Review by RubenEppink — APPROVED** [review]
lgtm

File: documentatie/Software-Guidebook/Software-Guidebook-WCAG-Tool.md [hunk 1]
```diff
@@ -18,38 +18,60 @@
...
+Most websites today aren’t properly programmed/configured to be accessible for everyone. 
+A blind person for example, must rely on the text-to-speech functionality to know and understand what content a website contains. 
+If a website isn’t programmed correctly for the text-to-speech functionality to work (properly), a blind person simply wouldn’t be able to use the website. 
+If a person isn’t able to use mouse and keyboard due to circumstances, would the website be able to utilize voice control to help this person?
 
-Because of these issues that occur on most websites, accesibility analists evaluate and judge websites based on how accessible they are using the Web Content Accessibility Guidelines (WCAG) requirements.
-Afterwards they send the company a report with any issues they found and a score.
+These are accessibility issues that occur on a lot of websites and leave users frustrated. 
+In the past people with handicaps or disabilities had to adapt to the environment, but in today’s world the digital environment should adapt to these people.
```
Comment (mjqnsen) [inline documentatie/Software-Guidebook/Software-Guidebook-WCAG-Tool.md:31]:
```suggestion
In the past, people with handicaps or disabilities had to adapt to the environment, but in today’s world the digital environment should adapt to these people.
```

File: documentatie/Software-Guidebook/Software-Guidebook-WCAG-Tool.md [hunk 2]
```diff
@@ -18,38 +18,60 @@
...
+- Account management
+
+What kind of features/functions are (very) significant?
+- Creating snapshots /w version control
+- Creating annotations on those snapshots
+- Collaboration between analysts
+
+Stakeholders:
+- Analyst as main character
+- Company (employee) as secondary character
```
Comment (mjqnsen) [inline documentatie/Software-Guidebook/Software-Guidebook-WCAG-Tool.md:74]:
```suggestion
- WCAG Analyst
- Website owner
```

**Review by mjqnsen — CHANGES_REQUESTED** [review]

**Review by mjqnsen — APPROVED** [review]

**Pull request merged**